docs/background/index.rst

```diff
@@ -27,11 +27,53 @@ TODO
 
 Hello World.
 
+.. _bg_models:
+
+Background Models
+=================
+
+The naming of the models in this section follows the convention from
+:ref:`dataformats_overview`.
+
+.. _background_3D:
+
+BACKGROUND_3D
+-------------
+
+`BACKGROUND_3D` is a background rate 3D cube (X, Y, energy) in
+units of per energy, per time, per solid angle. `X` and `Y` are
+given in detector coordinates `(DETX, DETY)`, a.k.a.
+`nominal system`. This is a tangential system to the instrument
+during observations.
+
+The `~CubeBackgroundModel` is used as container class for this model.
+It has methods to read, write and operate the 3D cubes.
+
+For the moment, only I/O and visualization methods are implemented.
+A test file is located in the `~gammapy-extra` repository
+(`bg_cube_model_test.fits <https://github.com/gammapy/gammapy-extra/blob/master/test_datasets/background/bg_cube_model_test.fits>`_).
+The file comes originally from the `~GammaLib` repository but has
+been slightly modified.
+
+TODO: the scripts produce some white canvases that I need to remove!
+
+An example script of how to read/write the files and perform some
+simple plots is given in the `examples` directory:
+
+.. plot:: ../examples/plot_background_model.py
+   :include-source:
+
+More complex plots can be easily produced with a few lines of code:
+TODO: mosaic/stack plots examples!!!
```

Comment:
Resolve this TODO.

I don't see the need for three plotting scripts, my suggestion would be to:
- Remove the plotting from the first code example ... just have that read and write.
- Combine the other two plotting scripts in one. Show three panels in one row (using `plt.subplots`), which show two images (one at very low and one at very high energy, so that they look different), and for the spectrum loop over a handful positions and draw the curves in the third panel.